discord/ext/converters/model.py

```diff
@@ -0,0 +1,57 @@
+import inspect
+import typing
+
+import discord
+from discord.ext.commands import converter, Command, Group, Context
+
+
+BASE_CONVERTERS = {
+    discord.CategoryChannel: converter.CategoryChannelConverter,
+    discord.Colour:          converter.ColourConverter,
+    discord.Emoji:           converter.EmojiConverter,
+    discord.Game:            converter.GameConverter,
+    discord.Invite:          converter.InviteConverter,
+    discord.Member:          converter.MemberConverter,
+    discord.Message:         converter.MessageConverter,
+    discord.PartialEmoji:    converter.PartialEmojiConverter,
+    discord.Role:            converter.RoleConverter,
+    discord.TextChannel:     converter.TextChannelConverter,
+    discord.User:            converter.UserConverter,
+    discord.VoiceChannel:    converter.VoiceChannelConverter,
+}
+
+
+class ConverterDict(dict):
+    def __init__(self):
+        super().__init__(BASE_CONVERTERS)
+    
+    def __setitem__(self, k: typing.Any, v: typing.Any):
+        if not (issubclass(v, converter.Converter) or inspect.isbuiltin(v)):
+            raise TypeError('Excepted value of type \'Converter\' or built-in, received %r' % v.__name__)
```

Comment:
```suggestion
            raise TypeError("Excepted value of type 'Converter' or built-in, received %r" % v.__name__)
```

Why are you still using Python 2 format syntax?